### DISCUSSION TITLE
feed

### DISCUSSION THREAD

**Discussion by arijit2002** [community/community#13502] [Other Feature Feedback, Questions, & Ideas] [answered]
pls don't make it a random social media..

**Comment by H3l1opy** [comment]
Concordo

**Comment by jfullstackdev** [comment, marked as answer]
there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
